deployments/bscmainnet_addresses.json

```diff
@@ -2,9 +2,23 @@
   "name": "bscmainnet",
   "chainId": "56",
   "addresses": {
-    "DefaultProxyAdmin": "0x6beb6D2695B67FEb73ad4f172E8E2975497187e4",
+    "DefaultProxyAdmin": "0x3f918bA9446552AB184C6Ffd2e2fcB1FA5ee1e59",
```

Comment:
`0x6beb6D2695B67FEb73ad4f172E8E2975497187e4` is the DefaultProxyAdmin
`0x3f918bA9446552AB184C6Ffd2e2fcB1FA5ee1e59` is the PSMProxyAdmin, imported from the core pool npm package

Reply:
I think we should keep `0x6b...` as DefaultProxyAdmin, and I would expect to have `0x3f...` as `PSMProxyAdmin`